{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "3689bcd1_5fb37d5d",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 11904
      },
      "writtenOn": "2022-04-04T14:16:37Z",
      "side": 1,
      "message": "recheck",
      "revId": "9b0e9db3636448d927bb0c371eee9311715f79db",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "6da572e4_57b11e21",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 28522
      },
      "writtenOn": "2022-04-05T09:09:23Z",
      "side": 1,
      "message": "recheck\n\nI wonder if the error in `requirements-tox-py36-check-uc` is legit. \nThe support of python 3.6 have been dropped in the setup.py of the os-testr project so I wonder if it impact the dependency resolver\nhttps://opendev.org/openstack/os-testr/commit/f8b7a194e1e3dd4cae02fd85b73020f33cd7c3b0",
      "revId": "9b0e9db3636448d927bb0c371eee9311715f79db",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "09964adb_1dc5f8be",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 28522
      },
      "writtenOn": "2022-04-05T09:09:58Z",
      "side": 1,
      "message": "022-04-04 14:23:11.754521 | ubuntu-bionic | ERROR: Could not find a version that satisfies the requirement os-testr\u003d\u003d\u003d3.0.0 (from versions: 0.1.0, 0.2.0, 0.3.0, 0.4.0, 0.4.1, 0.4.2, 0.5.0, 0.6.0, 0.7.0, 0.8.0, 0.8.1, 0.8.2, 1.0.0, 1.1.0, 2.0.0, 2.0.1)\n\n2022-04-04 14:23:11.755254 | ubuntu-bionic | ERROR: No matching distribution found for os-testr\u003d\u003d\u003d3.0.0\n\n2022-04-04 14:23:13.488370 | ubuntu-bionic | ERROR: invocation failed (exit code 1)\n\n2022-04-04 14:23:13.488694 | ubuntu-bionic | ERROR: could not install deps [-r/home/zuul/src/opendev.org/openstack/requirements/upper-constraints.txt]; v \u003d InvocationError(\u0027/home/zuul/src/opendev.org/openstack/requirements/.tox/py36-check-uc/bin/python -m pip install -r/home/zuul/src/opendev.org/openstack/requirements/upper-constraints.txt\u0027, 1)\n\n2022-04-04 14:23:13.488784 | ubuntu-bionic | py36-check-uc finish: getenv /home/zuul/src/opendev.org/openstack/requirements/.tox/py36-check-uc after 107.16 seconds\n\n2022-04-04 14:23:13.489102 | ubuntu-bionic | ___________________________________ summary ____________________________________\n\n2022-04-04 14:23:13.489180 | ubuntu-bionic | ERROR:   py36-check-uc: could not install deps [-r/home/zuul/src/opendev.org/openstack/requirements/upper-constraints.txt]; v \u003d InvocationError(\u0027/home/zuul/src/opendev.org/openstack/requirements/.tox/py36-check-uc/bin/python -m pip install -r/home/zuul/src/opendev.org/openstack/requirements/upper-constraints.txt\u0027, 1)\n\n2022-04-04 14:23:13.664445 | ubuntu-bionic | ERROR\n\n2022-04-04 14:23:13.664794 | ubuntu-bionic | {\n\n2022-04-04 14:23:13.664891 | ubuntu-bionic |   \"delta\": \"0:01:47.692099\",\n\n2022-04-04 14:23:13.664980 | ubuntu-bionic |   \"end\": \"2022-04-04 14:23:13.518784\",\n\n2022-04-04 14:23:13.665065 | ubuntu-bionic |   \"msg\": \"non-zero return code\",\n\n2022-04-04 14:23:13.665150 | ubuntu-bionic |   \"rc\": 1,\n\n2022-04-04 14:23:13.665233 | ubuntu-bionic |   \"start\": \"2022-04-04 14:21:25.826685\"\n\n2022-04-04 14:23:13.665316 | ubuntu-bionic | }",
      "parentUuid": "6da572e4_57b11e21",
      "revId": "9b0e9db3636448d927bb0c371eee9311715f79db",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    }
  ]
}